### DISCUSSION TITLE
how to set up domain in remote dev server ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [adonisjs/core#3892] [Help] [answered]
I installed fresh adonisjs app.

In welcome page I can see followings 

```edge
@entryPointStyles('app')
@entryPointScripts('app')
```

It interprets to this in html source view.

```html
<link rel="stylesheet" href="http://localhost:8080/assets/app.css" />
<script src="http://localhost:8080/assets/app.js" defer></script>

```

In local environments, it is fine. 

But when I upload the welcome page on remote server, it still use same domain  'localhost'. 

So it fails to load assets.

I tried with <code> {{ asset('assets/app.css') }} </code>  but same results.

I see there are 'entrypoints.json' and 'manifest.json' in /public/assets.

How can I change this domain without publishing as production status?

Should I edit 'entrypoints.json' and 'manifest.json' manually ? 

I could not find clues  in Deployment section of documentation.

**Comment by RomainLanz** [comment, marked as answer]
Hey @magic-thomas! 👋🏻 

You cannot simply output HTML and expect it to work remotely.
You have to build your application and deploy it.

Feel free to check our deployment guide.
📚 https://docs.adonisjs.com/guides/deployment#serving-static-assets

**Reply by magic-thomas** [reply]
So you mean no way without building production.

It is so uncomfortable for showing how it is going on to client. 

anyway thanks.
